core/src/main/java/com/linecorp/armeria/internal/DefaultTimeoutController.java

```diff
@@ -76,100 +84,166 @@ public void setTimeoutTask(TimeoutTask timeoutTask) {
      *
      * <p>Note that the {@link TimeoutTask} should be set via the {@link #setTimeoutTask(TimeoutTask)} or
      * the {@link #DefaultTimeoutController(TimeoutTask, EventLoop)} before calling this method.
+     *
+     * @return {@code true} if the timeout is scheduled.
+     *         {@code false} if the timeout has been scheduled, the timeout has been triggered already
+     *         or the {@link TimeoutTask} could not be scheduled now.
```

Comment:
Perhaps we can remove this, because it's already in the super class Javadoc?

Reply:
The `TimeoutTask` is only used for `DefaultTimeoutController`.
I will add a more general expression to describe why it could return `false`. 😀